benchmarks/decoders/benchmark_decoders.py

```diff
@@ -86,38 +86,72 @@ def get_consecutive_frames_from_video(self, video_file, numFramesToDecode):
 class TVNewAPIDecoderWithBackend(AbstractDecoder):
     def __init__(self, backend):
         self._backend = backend
+        self._print_each_iteration_time = False
         import torchvision  # noqa: F401
 
         self.torchvision = torchvision
 
     def get_frames_from_video(self, video_file, pts_list):
+        start = timeit.default_timer()
         self.torchvision.set_video_backend(self._backend)
         reader = self.torchvision.io.VideoReader(video_file, "video")
+        create_done = timeit.default_timer()
         frames = []
         for pts in pts_list:
             reader.seek(pts)
             frame = next(reader)
             frames.append(frame["data"].permute(1, 2, 0))
+        frames_done = timeit.default_timer()
+        if self._print_each_iteration_time:
+            del reader
```

Comment:
Are you intending to time how long it takes to do the deallocations of memory on the C++ side? Note that `del reader` on the Python side will only decrement a reference counter. You can make an explicit call to the garbage collector (see https://docs.python.org/3/library/gc.html#gc.collect), but that's going to do a full collection of all garbage.

Reply:
This code was for my own debugging/edification and is turned off by default. A private variable controls it.

I can delete it if you want. It's developer-only code -- not user-facing.

Let me know.

I timed the code btw -- when using pytorch to time it, the aggregate profile doesn't change when I call `del decoder` on it. It's just that if you print the iters, you can see a timing difference once in every few iters.

Reply:
I think the benchmark is easier to understand without it, honestly. If we keep it, then we need a comment explaining that we know this would happen anyway when the function exits, and this does not actually cause the GC to happen, but doing this here before the function exists allows us to sometimes see the cost of GC.

Reply:
I removed the del line now